File: cocoadocs.lbaction/Contents/Scripts/default.rb
#!/usr/bin/ruby
require 'open-uri'
require 'JSON'

query = ARGV[0]
output = Array.new
results = JSON.parse(open("http://search.cocoapods.org/api/v1.5/pods/search?query=#{query}").read)

results.each do |item|
    hash = Hash.new
    hash["title"] = item["id"]+" "+item["version"]
    hash["subtitle"] = item["summary"]
    hash["url"] = "http://cocoadocs.org/docsets/#{item['id']}/#{item['version']}"
    output.push(hash)
end

puts output.to_json

File: cocoadocs.lbaction/Contents/Scripts/default.py
#!/usr/bin/env python
# import requests
from urllib import FancyURLopener
import json
import sys

query = sys.argv[1]

class MyOpener(FancyURLopener):
    version = 'AppleWebKit/537.11 (KHTML, like Gecko) Safari/537.11'
    
site= "http://search.cocoapods.org/api/v1/pods.flat.hash.json?query={0}".format(query)
o = MyOpener()
results = json.loads(o.open(site).read())
output = []

for item in results:
    itemOut = {}
    itemOut["title"] = "{0}{1}".format(item["id"],item["version"])
    itemOut["subtitle"] = item["summary"]
    itemOut["url"] = "http://cocoadocs.org/docsets/{0}/{1}".format(item["id"], item["version"])
    output.append(itemOut)
    
print json.dumps(output)